android/src/toga_android/widgets/label.py

```diff
@@ -66,9 +66,12 @@ def set_alignment(self, value):
         if not self.native.getLayoutParams():
             return
 
-        # Justified text wasn't added until Android O (SDK 26)
-        if value == JUSTIFY and Build.VERSION.SDK_INT >= Build.VERSION_CODES.O:
-            self.native.setJustificationMode(LineBreaker.JUSTIFICATION_MODE_INTER_WORD)
-        else:
-            self.native.setJustificationMode(LineBreaker.JUSTIFICATION_MODE_NONE)
+        # Justified text wasn't added until API level 26.
+        if Build.VERSION.SDK_INT >= 26:
+            self.native.setJustificationMode(
+                Layout.JUSTIFICATION_MODE_INTER_WORD
+                if value == JUSTIFY
+                else Layout.JUSTIFICATION_MODE_NONE
```

Comment:
These constants were added to the `Layout` class in API level 26, then copied to the `LineBreaker` class in API level 29. So we don't actually need to reference `LineBreaker` at all.